### PR TITLE
disable the internal s3sdk multi part copy logic

### PR DESCRIPTION
**Pull request by icewind1991**
We have our own logic so just set the threshold in the library to 2^64-1

### CODE REVIEW

File: lib/private/Files/ObjectStore/S3ObjectTrait.php [hunk 1]
```diff
@@ -212,7 +212,8 @@ public function copyObject($from, $to, array $options = []) {
 			$copy->copy();
 		} else {
 			$this->getConnection()->copy($this->getBucket(), $from, $this->getBucket(), $to, 'private', array_merge([
-				'params' => $this->getSSECParameters() + $this->getSSECParameters(true)
+				'params' => $this->getSSECParameters() + $this->getSSECParameters(true),
+				'mup_threshold' => PHP_INT_MAX,
```
Comment (Fenn-CS) [inline lib/private/Files/ObjectStore/S3ObjectTrait.php:216]: The threshold is set to max? What's the default?

**Review by nfebe — APPROVED** [review]
Looks okay. 

_However, if default is max, then it might be unnecessary?_

File: lib/private/Files/ObjectStore/S3ObjectTrait.php (hunk 1 above)
Comment (icewind1991) [reply lib/private/Files/ObjectStore/S3ObjectTrait.php:216]: 5GB I believe

**Review by Altahrim — APPROVED** [review]

**Pull request merged**